static/js/apps/nl_interface/query_history.tsx

```diff
@@ -41,14 +41,16 @@ export function QueryHistory(props: QueryHistoryProps): JSX.Element {
       {!_.isEmpty(history) && (
         <div className="container nl-history">
           <h3>Or try one of these recent queries:</h3>
-          {history.map((query, i) => {
+          {history.map((queries, i) => {
             return (
               <div
                 className="history-item"
                 key={i}
-                onClick={() => props.onItemClick(query)}
+                onClick={() => props.onItemClick(history[i])}
+                title={queries.join(", ")}
```

Comment:
Mostly out of curiosity, where the `;` join happening?

Reply:
this is just for display purposes. so a long hover over the text will show you the full query (with comma separation).

`onItemClick(history[i])` sends the full array of queries for that "session" to the parent which does the query-by-query handling.

Reply:
thanks!